recipes/sassc/all/conanfile.py

```diff
@@ -1,8 +1,11 @@
-from conans import ConanFile, AutoToolsBuildEnvironment, tools, MSBuild
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, replace_in_file, chdir, save
+from conan.tools.microsoft import is_msvc
+from conans import AutoToolsBuildEnvironment, tools, MSBuild
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
1.47.0 should be enough here.